packages/block-editor/src/components/height-control/index.js

```diff
@@ -144,13 +149,14 @@ export default function HeightControl( {
 	};
 
 	return (
-		<fieldset className="block-editor-height-control">
-			<BaseControl.VisualLabel as="legend">
+		<div className="block-editor-height-control" id={ id }>
+			<BaseControl.VisualLabel as="label" for={ inputId } id={ labelId }>
```

Comment:
I guess this `for` should have been `htmlFor`. Getting this in the condole:
````
Warning: Invalid DOM property `for`. Did you mean `htmlFor`?
at label
at div
at HeightControl
etc...
```

Reply:
Yup - I must have let my guard down knowing that this PR had been reviewed before. Thank you for pointing it out